.test/config.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+imageTests+=(
+	[openjdk]='
+		java-hello-world
+		java-uimanager-font
+		java-ca-certificates
+		java-ca-certificates-update
+	'
+)
```

Comment:
I don't love this feature (as I've now noted in https://github.com/docker-library/docs/pull/2338#issuecomment-1593765498), but this should be able to be written more simply / in a way that's less likely to miss tests we might add in the future like this:

```suggestion
imageTests[openjdk]+='
	java-ca-certificates-update
'
```

(it's just a standard shell "whitespace separated list" because you can't embed bash arrays inside bash associative arrays directly, so it could even just be `...+=' test-name'` if you wanted something slightly more minimal)

This should work because our wrapper explicitly sources the upstream `config.sh` first (for exactly this purpose).